### PR TITLE
Add search bar to the custom file manager

### LINKED ISSUE #403: Add search/filter bar for the file browser
It would be helpful to have a search bar for the file browser so people can look for particular files.

### PR DESCRIPTION
**Pull request by bruyeret**
Close #403

### CODE REVIEW

**Review by arjunrajlab — APPROVED** [review]
Overall, love it! I think it would be nice to have a little text that says "Search" next to the search bar to make it a little more discoverable, because right now it's a little hard to notice (see screenshot). But the functionality is great!

image

File: src/components/CustomFileManager.vue [hunk 1]
```diff
@@ -156,15 +178,41 @@ export default class CustomFileManager extends Vue {
...
+      case "file":
+        return "file";
+      case "folder":
+        return "folder";
+      case "item":
+        return "file";
+      case "user":
+        return "user";
+    }
+  }
```
Comment (luciemac) [inline src/components/CustomFileManager.vue:210]:
```
switch (selectable._modelType) {
      case "file":
      case "item":
        return "file";
      case "folder":
        return "folder";
      case "user":
        return "user";
    }
```

**Comment by luciemac** [comment]
then, LGTM

**Comment by bruyeret** [comment]
There is a loupe icon but it is white even in light mode, I don't know why
The icon to the right of the search bar is dark in light mode
I added the "search" text anyway @arjunrajlab 

image

image

Also to switch between dark mode and light mode, I had to open a dataset now that the button is in the settings panel, shouldn't it be also in home page?

**Comment by zjniu** [comment]
LGTM!

**Comment by luciemac** [comment]
LGTM

**Pull request merged**